src/ai/defaultai.cc

```diff
@@ -6872,6 +6918,11 @@ void DefaultAI::gain_building(Widelands::Building& b, const bool found_on_load)
 			if (bo.is(BuildingAttribute::kPort)) {
 				++num_ports;
 			}
+			if (!found_on_load) {
+				// recalculate distance ASAP
+				const Time& gametime = game().get_gametime();
```

Comment:
this line is superfluous due to line 6835

Reply:
well our checks are much harder then previous now, however this is increasing code quality.